platform/src/server/http/__tests__/http_server.test.ts

```diff
@@ -47,8 +53,8 @@ test('200 OK with body', async () => {
 
   await server.start(config);
 
-  await supertest((server as any).server)
-    .get('/foo')
+  await supertest(getServerListener(server))
+    .get('/foo/')
```

Comment:
__note:__ all these added slashes here and in tests below because of the fact that ExpressJS has disabled so called "strict routing" by default (`/foo` and `/foo/` are treated the same by the router in non-strict mode), but not Hapi.